account_invoice_supplierinfo_update_discount/wizard/wizard_update_invoice_supplierinfo_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2016-Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api, fields
```

Comment:
Import statement should be in alphabetical order

Reply:
Done.